src/main/scala/cromwell/engine/WorkflowManager.scala

```diff
@@ -0,0 +1,83 @@
+package cromwell.engine
+
+import java.util.UUID
+
+import akka.actor.{Actor, ActorSystem, ActorRef}
+import akka.pattern.pipe
+import scala.concurrent.ExecutionContext.Implicits.global
+import cromwell.binding.{WdlBinding, WdlInputs, Wdl}
+import cromwell.engine.WorkflowActor.Start
+import cromwell.engine.WorkflowManagerActor.{SubmitWorkflow, WorkflowStatus}
+import cromwell.engine.backend.Backend
+import cromwell.engine.backend.local.LocalBackend
+
+import scala.concurrent.Future
+
+/**
+ * Abstract notion of a workflow manager. Mainly exists to allow for actor-system-free unit testing of basic concepts
+ */
+trait WorkflowManager {
+  type Workflow
+
+  case class ManagedWorkflow(id: WorkflowId, workflow: Workflow)
+
+  def backend: Backend
```

Comment:
The person who made the above comment is an idiot. He just described a 'val' which conveniently exists in the language.